examples/dsk_LoRaE5_Debuging/dsk_LoRaE5_Debuging.ino

```diff
@@ -3,9 +3,9 @@
 
 Disk91_LoRaE5 lorae5(&Serial); // Where the AT command and debut traces are printed
 
-uint8_t deveui[] = { 0xDC, 0xXX, 0xXX, 0xXX, 0xXX, 0xXX, 0xXX, 0xEE };
-uint8_t appeui[] = { 0xCD, 0xXX, 0xXX, 0xXX, 0xXX, 0xXX, 0xXX, 0xAD };
-uint8_t appkey[] = { 0x2A, 0xXX, 0xXX, 0xXX, 0xXX, 0xXX, 0xXX, 0xA1, 0xXX, 0xXX, 0xXX, 0xXX, 0xXX, 0xXX, 0xXX, 0x43 };
+AT+ID=DevEui,"2CF7FXXXXXX0A49F"
```

Comment:
- I don't want to use sting for keys because it's not efficient in term for memory, because it's not good in terms of security as the string are too easy to search and found on a binary. This can't be made as an example.
- compilation issue is a good practice to make sure the user will fill it with the right values.